code/controllers/subsystem/vote.dm

```diff
@@ -142,13 +142,13 @@ SUBSYSTEM_DEF(vote)
 				var/datum/map_config/VM = config.maplist[SHIP_MAP]["Combat Patrol Base"]
 				SSmapping.changemap(VM, SHIP_MAP)
 			if(GLOB.master_mode != .)
+				SSticker.save_mode(.)
 				var/datum/game_mode/current_gamemode = config.pick_mode(GLOB.master_mode)
 				if((current_gamemode.flags_round_type & MODE_SPECIFIC_SHIP_MAP) && !SSticker.HasRoundStarted())
```

Comment:
this code is highly non generic and a bit confusing, please make it proper

Reply:
less guh'd, let me know if it's still scuff though.